aula-css/exercicios/photo-grid/aula-1/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Monalisa's Photo Grid</title>
+    
+    <link rel="stylesheet" href="style.css"/>
+</head>
+
+<body>
+    <main>
+        <div class="apresentation">
+            <h1><strong>Monalisa Scalco's Photo Grid :)</strong></h1>
```

Comment:
```suggestion
            <h1>Monalisa Scalco's Photo Grid :)</h1>
```

O `<h1>` por padrão já exibe o texto em negrito, além de no quesito relevância ser superior ao `<strong>`, então não há motivo para esse uso.